### ISSUE TITLE
Additional errors on Home Assistant 2022.7

### ISSUE THREAD

**Issue by robloh** [HASwitchPlate/openHASP-custom-component#102]
## Version of the custom_component
0.6.5

## Configuration

```yaml

openhasp:
  plate:
    objects:
      - obj: "p1b10"
        properties:
          "text": "{{ states('sensor.date_time') }}"
```

## Describe the bug
More things openHASP component is using have been deprecated and will be removed by 2022.10 Home Assistant release

## Debug log

```text

2022-07-06 16:53:55 WARNING (MainThread) [homeassistant.components.number] custom_components.openhasp.number is setting deprecated attributes on an instance of NumberEntityDescription, this is not valid and will be unsupported from Home Assistant 2022.10. Please report it to the custom component author.

2022-07-06 16:53:55 WARNING (MainThread) [homeassistant.components.number] custom_components.openhasp.number::HASPNumber is overriding deprecated methods on an instance of NumberEntity, this is not valid and will be unsupported from Home Assistant 2022.10. Please report it to the custom component author.

2022-07-06 16:53:55 WARNING (MainThread) [homeassistant.components.number] Entity None (<class 'custom_components.openhasp.number.HASPNumber'>) is using deprecated NumberEntity features which will be unsupported from Home Assistant Core 2022.10, please report it to the custom component author.
```

**Comment by fvanroie** [comment]
These are warnings, not errors. I'm sure they will be fixed by the time 2022.10 comes around.

**Comment by dgomes** [comment]
Fixed in the last commit, should be available in the next release
